lib/spike-detector.js

```diff
@@ -0,0 +1,101 @@
+var moment = require('moment');
+var _ = require('underscore');
+var debug = require('debug')('spikeD');
+var events = require('events');
+var eventEmitter;
+
+module.exports = function (options) {
+
+  function SpikeD() {
+    eventEmitter = new events.EventEmitter();
+    eventEmitter.on('checkForSpike', this.checkForSpike.bind(this));
```

Comment:
looks like this doesn't belong here. we should use `.emit` and emit the event on one of the `increments` once we detect the spike. the `.on` should be in 'customs'